blocks/filter-group/filter-group.css

```diff
@@ -0,0 +1,81 @@
+/* !important added to override system styles to maintain cutoff 
+effect on filter buttons to indicate horizontal scroll */
+.filter-group-wrapper {
+  padding-right: 0 !important;
+
+  @media (min-width: 76rem) {
```

Comment:
is this breakpoint just for this component? where are we getting this number?

Reply:
Yes, this breakpoint is specific to this component, I'm removing some right padding to accommodate the cutoff effect of these buttons and it needed to be reset at this breakpoint to prevent the component from shifting when reducing the page size.

Reply:
sweet!